Source.cpp

```diff
@@ -0,0 +1,105 @@
+#include "Header.h"
+
+Bag::Bag(string description, int severity, float deadline, string status, string assegnee)
+{
+    this->description = description;
+    this->deadline = deadline;
+    this->assegnee = assegnee;
+    this->severity = severity;
+    this->status = status;
+    //cout << "Description - " << description << "\nDifficalty - " << severity << "\nDeadline - " << deadline << "\nStatus - " << status << "\nAssegnee  - " << assegnee;
+}
+Bag::Bag(Bag& B) {
+    this->previous = NULL;
+    this->next = NULL;
+    this->description = B.description;
+    this->severity = B.severity;
+    this->deadline = B.deadline;
+    this->status = B.status;
+    this->assegnee = B.assegnee;
+}
+void Bag::showSomething() const {
+    cout << "Description - " << description << "\nDifficalty - " << severity << "\nDeadline - " << deadline << "\nStatus - " << status << "\nAssegnee  - " << assegnee;
+}
+
+
+void BackLog::insert(Bag* give1) {
+    Bag* give = new Bag(*give1);
+    if (begin == NULL) {
+        begin = give;
+        end = give;
+        return;
+    }
+    if (begin == end) {
+        begin->next = give;
+        end = begin->next;
+        end->previous = begin;
+        return;
+    }
+    end->next = give;
+    end->next->previous = end;
+    end = end->next;
+}
+
+void BackLog::show() {
+    Bag* kol = begin;
+    while (kol != NULL) {
+        kol->showSomething();
+        kol = kol->next;
+    }
+}
+
+BackLog :: ~BackLog() {
+    Bag* kol = begin;
+    while (kol != NULL) {
+        Bag* temp = kol;
+
+        kol = kol->next;
+
+        temp->showSomething();
+        delete temp;
+    }
+}
+
+void BackLog::showByStatus() {
+
+    Bag* kol = begin;
+    while (kol != NULL) {
+        if (kol->status == "Resolved") {
+            kol->showSomething();
+        }
+        kol = kol->next;
+    }
+}
+void BackLog::shortBySeverity() {
+    Bag* kol = begin;
+    while (kol != end) {
+        Bag* kol2 = kol->next;
+        Bag* smallnumber = kol;
+        while (kol2 != NULL) {
+            if (kol2->severity < smallnumber->severity) {
+                smallnumber = kol2;
+
+            }
+            kol2 = kol2->next;
+        }
+        if (smallnumber != kol)
+        {
```

Comment:
Please, move this bracket to previous line or move bracket a new line for every block

Reply:
Fixed